fetch_videos.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
```

Comment:
Make sure to include copyright headers in this file

Reply:
Done